vuu-ui/packages/vuu-filters/src/filter-clause/FilterClauseEditor.tsx

```diff
@@ -60,62 +59,13 @@ export const FilterClauseEditor = ({
     filterClause,
     onCancel,
     onChange,
-    tableSchema,
+    columnDescriptors,
   });
 
-  const getInputElement = useCallback(() => {
-    if (selectedColumn === null || operator === undefined) {
-      return null;
-    }
-    switch (selectedColumn?.serverDataType) {
-      case "string":
-      case "char":
-        return (
-          <TextInput
-            InputProps={InputProps}
-            className={cx(`${classBase}Field`, `${classBase}Value`)}
-            column={selectedColumn}
-            data-field="value"
-            filterClause={filterClause}
-            onDeselect={onDeselectValue}
-            onInputComplete={onChangeValue}
-            operator={operator}
-            suggestionProvider={suggestionProvider}
-            table={table}
-            value={value as string | string[]}
-          />
-        );
-      case "int":
-      case "long":
-      case "double":
-        return (
-          <NumericInput
-            InputProps={InputProps}
-            className={cx(`${classBase}Field`, `${classBase}Value`)}
-            column={selectedColumn}
-            filterClause={filterClause}
-            onInputComplete={onChangeValue}
-            operator={operator}
-          />
-        );
-      case undefined:
-        console.log("returning undefined");
-        return undefined;
-      default:
-        console.log("returning unsupported");
-        return null;
-    }
-  }, [
-    selectedColumn,
-    operator,
-    InputProps,
-    filterClause,
-    onDeselectValue,
-    onChangeValue,
-    suggestionProvider,
-    table,
-    value,
-  ]);
+  const columns = useMemo(
+    () => Object.values(columnDescriptors),
```

Comment:
One consequence of switching from tableSchema.columns to tableConfig.columns is that whereas previously we could add any column from the Vuu table to our filter, now we can only include columns actually present in our Table (which includes hidden columns).   
I think this would probably be the expected behaviour (so an improvement) , though its worth at least thinking about.

Reply:
Ahh good to know this!